public/app/features/alerting/unified/components/rule-editor/AlertRuleForm.tsx

```diff
@@ -304,6 +315,7 @@ export const AlertRuleForm: FC<Props> = ({ existing, prefill }) => {
                 onDataChange={checkAlertCondition}
                 asyncDefaultQueries={defaultDsAndQueries.queries}
                 asyncDataSource={defaultDsAndQueries.ds}
+                defaults={Boolean(defaultsInQueryParams)}
```

Comment:
`prefill` is already a prop that takes a `boolean` and indicates if the form was prefilled from a set of values, we don't really need a new prop since we can just pass 

`prefill={Boolean(prefill) || Boolean(defaultsInQueryParams)}`

Reply:
💯